### DISCUSSION TITLE
Cloning error

### DISCUSSION THREAD

**Discussion by westfinn** [community/community#40859] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i'm cloning into a remote repo and it fails.  what is going wrong and how do i fix.    i'm new to git.

git clone https://github.com/supermerill/SuperSlicer.git
Cloning into 'SuperSlicer'...
remote: Enumerating objects: 204888, done.
remote: Counting objects: 100% (14/14), done.
remote: Compressing objects: 100% (14/14), done.
**error: RPC failed; curl 56 GnuTLS recv error (-54): Error in the pull function.
error: 1108 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet**
fatal: early EOF
fatal: index-pack failed

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @westfinn ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
